tests/framework/stats/core.py

```diff
@@ -73,7 +73,9 @@ def run_exercise(self, fail_fast=False) -> Result:
                     raws = pipe.consumer.ingest(iteration, data)
                     if raws is not None:
                         dimensions = self.custom.copy()
-                        dimensions["test"] = self.metrics_test
+                        test = tag.split("/")[-1]
+                        dimensions["test"] = test
```

Comment:
hmm I did this to kind-of guarantee uniqueness, whereas in the others we named the perf test by hand, but it makes sense to be consistent with the other metrics

Reply:
The problem I ran into this only having the metrics name was that the "tag" part was the only thing that contained the pytest parameters (e.g. payload length and such). For for the raw metrics, I couldn't map them to their parameters :(